iconsdk/libs/in_memory_zip.py

```diff
@@ -59,7 +59,7 @@ def zip_in_memory(self, _path: str):
         try:
             # when it is a zip file
             if path.isfile(_path):
-                zf = ZipFile(_path, 'r', ZIP_DEFLATED, False)
+                zf = ZipFile(_path, 'r', ZIP_DEFLATED, False, compresslevel=9)
```

Comment:
This block is just for reading a existing zip file and writing its content into memory. Thus we don't need to specify `compresslevel` here.